### DISCUSSION TITLE
【功能性建议】cfw的iplist可否不使用ip段，只使用指定ip？

### DISCUSSION THREAD

**Discussion by chenxuuu** [SeaHOH/GotoX#160] [点子]
我不太清楚这里要用ip段的原因

但是一般来说，筛选下来速度快的只有那么十几个ip，这里只能用整个ip段速度和不配置前基本没什么区别。。。

**Comment by SeaHOH** [comment]
响应速度的使用感受确实没太多区别，这么做的原因是设置的 IP 不会一直有效，某些时候可能会没法使用，增加有后备就比较保险。

参考 https://github.com/SeaHOH/GotoX/issues/158#issuecomment-699442961 说明修改源码。

**Comment by SeaHOH** [comment]
https://github.com/SeaHOH/GotoX/issues/158#issuecomment-699442961 给出的方法有误，现已添加控制选项。
@k79e
